b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/getSentDataRequestOptions.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import config from '../../../config.json';
+
+function sentDataHeader(session) {
+    const headers = {
+        "accept": "application/scim+json",
+        "content-type": "application/scim+json",
+        "authorization": "Bearer " + session.accessToken,
+        "access-control-allow-origin": config.WSO2IS_CLIENT_URL
+    }
+    return headers;
+}
+
+function getSentDataRequestOptions(session, method, body) {
+    const request = {
+        method: method,
+        headers: sentDataHeader(session),
+        body: JSON.stringify(body)
+    }
+    return request;
+}
+
+module.exports = { getSentDataRequestOptions }
```

Comment:
Shall we add a new line here. Check the other places as well

Reply:
This was fixed with a later PR,

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-app/data-access/data-access-common-api-util/src/lib/apiUtil/api/apiRequestOptions.ts

### Commit
- https://github.com/wso2/samples-is/commit/a4ef8e350763d499a47398c49f061c7176f249ac#diff-968532fdc2f70738a9cbab5b80b9bd297eb489f158eea4fb3dce0db7062b4545